packages/design-system/src/scalars/components/toggle/toggle.test.tsx

```diff
@@ -1,75 +1,97 @@
 import React from "react";
-import { render, screen, fireEvent } from "@testing-library/react";
-import { describe, it, vi } from "vitest";
-import { ToggleField } from "./toggle-field";
+import { render, fireEvent, screen } from "@testing-library/react";
+import { Toggle } from "./toggle";
+import { vi } from "vitest";
 
 describe("Toggle Component", () => {
-  const mockOnChange = vi.fn();
+  const defaultProps = {
+    onChange: vi.fn(),
+    disabled: false,
+    checked: false,
+    required: false,
+  };
+
   beforeEach(() => {
     vi.clearAllMocks();
   });
 
-  it("should match snapshot", () => {
-    const { asFragment } = render(<ToggleField />);
-    expect(asFragment()).toMatchSnapshot();
-  });
+  describe("Functionality", () => {
```

Comment:
the description for this component should be more descriptive or direct, "functionality" doesn't lead to what actually this component is

Reply:
you are rigth, done